osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -127,6 +124,9 @@ public IBeatmap GetPlayableBeatmap(RulesetInfo ruleset)
             foreach (var obj in converted.HitObjects)
                 mod.ApplyToHitObject(obj);
 
+            // Post-process
+            rulesetInstance.CreateBeatmapProcessor(converted)?.PostProcess();
```

Comment:
this change not only caused a regression in catch behaviour (which i addressed) but also osu! and potentially other modes.

these kinds of changes need to be well tested and well documented going forwards.

![image](https://user-images.githubusercontent.com/191335/41921571-207ddb16-799e-11e8-95c6-7bc5a75a0d7b.png)